Documentation/Books/Manual/DataModeling/Views/README.md

```diff
@@ -2,42 +2,97 @@ JavaScript Interface to Views
+accessing views is discouraged. Views share their namespace with collections,
+so there cannot exist a view and a collection with the same name in the same
+database.
 
-### View
-`db._view(view-name)`
+Usage
+-----
 
-A view is created by a ["db._createView"](DatabaseMethods.md) call. The returned
-object may then be used via the [exposed methods](ViewMethods.md).
+Here follow some basic usage examples. More details can be found in the
+following chapters:
+- [Database Methods for Views](DatabaseMethods.md)
+- [View Methods](ViewMethods.md)
+- [ArangoSearch Views](../Views/README.md)
 
-For example: Assume that the
-[view identifier](../../Appendix/Glossary.md#view-identifier) is *7254820* and
-the name is *demo*, then the view can be accessed as:
+Create a view with default properties:
 
-    db._view("demo")
+    @startDocuBlockInline viewUsage_01
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_01}
+    ~ db._create("colA");
+    ~ db._create("colB");
+    view = db._createView("myView", "arangosearch", {});
+    ~ addIgnoreCollection("colA");
+    ~ addIgnoreCollection("colB");
+    ~ addIgnoreView("myView");
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_01
 
-If no view with such a name exists, then *null* is returned.
+Get this view again later by name:
 
-### Create
-`db._createView(view-name, view-type, view-properties)`
+    @startDocuBlockInline viewUsage_02
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_02}
+    view = db._view("myView");
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_02
 
-This call will create a new view called *view-name*. This method is a database
-method and is documented in detail in
-[Database Methods](DatabaseMethods.md#create)
+Get the view properties:
 
-### View Types
+    @startDocuBlockInline viewUsage_03
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_03}
+    view.properties();
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_03
 
-The currently supported view implementation is: **arangosearch** as described in
-[ArangoSearch View](../../Views/ArangoSearch/README.md).
+Set a view property:
+
+    @startDocuBlockInline viewUsage_04
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_04}
+    view.properties({cleanupIntervalStep: 12});
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_04
+
+Add a link:
+
+    @startDocuBlockInline viewUsage_05
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_05}
+    view.properties({links: {colA: {includeAllFields: true}}});
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_05
+
+Add another link:
+
+    @startDocuBlockInline viewUsage_06
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_06}
+    view.properties({links: {colB: {fields: {text: {}}}}});
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock viewUsage_06
+
+Remove the first link again:
+
+    @startDocuBlockInline viewUsage_07
+    @EXAMPLE_ARANGOSH_OUTPUT{viewUsage_07}
+    view.properties({links: {colA: null}});
```

Comment:
must be `view.properties({links: {colA: null}}, true);`
Otherwise it will be treated as **full** update